browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -3108,6 +3108,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 		var separatorPos = elementId.indexOf(':'); // delete menuId
 		if (separatorPos > 0)
 			elementId = elementId.substr(0, separatorPos);
+		var isEditContainer = document.getElementById(rawId);
```

Comment:
best to use naming `isXXXX` for boolean values

Reply:
I think we shouldn't add such logic to the function "removeMenuId" (it's not obvious that it will add something for an element, especially that it will no longer point to the root node).

Can we try to adjust executeAction instead? I belive `case 'setText':` needs some adjustments.